custom_components/wibeee/nest.py

```diff
@@ -4,6 +4,7 @@
 
 from homeassistant.components.network import async_get_source_ip
 from homeassistant.components.network.const import PUBLIC_TARGET_IP
+from homeassistant.config_entries import ConfigEntry
```

Comment:
Let's please keep this module independent of the config flow. That is, pass in the primitive str/int values instead of the ConfigEntry.

Reply:
That was my first approach, but since I had that error about the parameters I changed it to that, just to try.
Also I wanted to add an advanced configuration option to add the new settings, but couldn't find how to do it :upside_down_face: